lib/core/enhanceError.js

```diff
@@ -16,12 +16,12 @@ module.exports = function enhanceError(error, config, code, request, response) {
     error.code = code;
   }
 
-  error.request = request;
+  error.request=request;
   error.response = response;
   error.isAxiosError = true;
 
-  error.toJSON = function() {
-    return {
+  error.toJSON = function( ) {
+    return{
```

Comment:
format: style mismatch:
```suggestion
    return {
```

`39615007` ␣ should be inserted at column 11.